packages/hdwallet-ledger/src/ethereum.ts

```diff
@@ -72,7 +111,7 @@ export async function ethSignTx(transport: LedgerTransport, msg: ETHSignTx): Pro
     v: parseInt(v, 16),
     r: '0x' + r,
     s: '0x' + s,
-    serialized: '0x' + toHexString(tx.serialize())
+    serialized: '0x' + core.toHexString(tx.serialize())
```

Comment:
```suggestion
    serialized: `0x${core.toHexString(tx.serialize())}`
```

Reply:
This syntax is spread throughout the whole code base. Not going to worry about this one spot at this time.